### PR TITLE
Ccpa

### PR DESCRIPTION
**Pull request by LauraDuRy**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: src/routes/blog/post/ccpa-vs-gdpr/+page.markdoc [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+---
+When you build your application, one of the first things you need to set up is your database and authentication. In other words, you're handling and storing user data. But with this data comes great responsibility. 
+
+If you’re here, you probably want to understand how to handle this responsibility and manage user data properly. In this blog, we’ll break down some of the major compliance standards like **California Consumer Privacy Act (CCPA)** and the **General Data Protection Regulation (GDPR),**  highlight their differences, and guide you on how to stay compliant with these regulations.
+
+# Overview of CCPA vs GDPR
+
+## What is GDPR?
+
+The **General Data Protection Regulation [GDPR](https://appwrite.io/docs/advanced/security/gdpr)is a comprehensive **data protection** law that came into effect on May 25, 2018, across the European Union (EU). It aims to protect the **personal data** of EU citizens and harmonize privacy laws across Europe. GDPR applies to all businesses, regardless of location, that process the personal data of EU residents. Any developers building applications for EU residents must be GDPR-compliant.
```
Comment (adityaoberai) [inline src/routes/blog/post/ccpa-vs-gdpr/+page.markdoc:20]: Screenshot_20240919_153537_Chrome

There are unnecessary asterisks and a missing space here

File: src/routes/blog/post/ccpa-vs-gdpr/+page.markdoc [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+## What is CCPA?
+
+The **California Consumer Privacy Act [CCPA](https://oag.ca.gov/privacy/ccpa), enacted on January 1, 2020, is a state-level privacy law in the United States that applies to residents of California. The **CCPA for developers** is often considered the most significant U.S. privacy regulation, giving Californians more control over their personal data and how businesses handle it. Developers looking to build **CCPA-compliant apps** must implement features that allow users to exercise their rights under the CCPA.
```
Comment (adityaoberai) [inline src/routes/blog/post/ccpa-vs-gdpr/+page.markdoc:24]: Screenshot_20240919_153645_Chrome

Unnecessary asterisks

**Review by adityaoberai — CHANGES_REQUESTED** [review]

File: src/routes/blog/post/ccpa-vs-gdpr/+page.markdoc (hunk 1 above)
Comment (LauraDuRy) [reply src/routes/blog/post/ccpa-vs-gdpr/+page.markdoc]: fixed

File: src/routes/blog/post/ccpa-vs-gdpr/+page.markdoc (hunk 2 above)
Comment (LauraDuRy) [reply src/routes/blog/post/ccpa-vs-gdpr/+page.markdoc]: fixed

**Review by adityaoberai — APPROVED** [review]

**Pull request merged**